src/main/java/org/aksw/simba/squirrel/analyzer/impl/GuessMimeType.java

```diff
@@ -0,0 +1,51 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+import java.io.File;
+import java.io.FileInputStream;
+import org.aksw.simba.squirrel.analyzer.Guess;
+import org.apache.tika.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GuessMimeType implements Guess {
+	private static final Logger LOGGER = LoggerFactory.getLogger(RDFAnalyzer.class);
+
+	public String guess(File data) {
+		String mimeType = null;
+		try {
+			FileInputStream inputStream = new FileInputStream(data.getAbsolutePath());
+			String fileContent = IOUtils.toString(inputStream);
+			if (fileContent.toString().isEmpty()) {

```

Comment:
Why are you calling `toString()` all the time? It is already a String... :thinking: